src/lib/types/index.js

```diff
@@ -21,9 +20,9 @@ export type Values = {
 }
 
 export type event = {
-    preventDefault: Function,
-    target: {
-      id: string,
-      value: string
-    }
+  preventDefault: Function,
```

Comment:
Since this is a browser default function leaving this as 'Function' type is completely fine.